packages/lodestar/src/network/encoders/response.ts

```diff
@@ -112,9 +122,25 @@ export function eth2ResponseDecode(
           }
         }
       }
+      if(status !== RpcResponseStatus.SUCCESS) {
```

Comment:
should we throw error here to mark an unsuccessful req/resp? that would make it the same treatment to others